### PR TITLE
Normalize build

### PR DESCRIPTION
**Pull request by jafcrocker**
- Fix unit tests
- Bump development version
- Remove obsolete image-building sources and targets

Note - Jenkins jobs changes to support this PR:
- Build action is ```make clean test wheel```
- The artifacts are in ```dist/*.whl``` instead of ```*.whl``` 
- Job builds from the master (instead of develop) branch

### CODE REVIEW

File: servicemigration/context.py [hunk 1]
```diff
@@ -41,7 +41,10 @@ def __init__(self, filename=None):
         else:
             infile = os.environ["MIGRATE_INPUTFILE"] if "MIGRATE_INPUTFILE" in os.environ else None
 
-        self._client = ControlPlaneClient(**getConnectionSettings())
+	if ZenUtils:
```
Comment (tgjonestx) [inline servicemigration/context.py]: Is this a functional change unrelated to cleanup, or something you needed to run the `test` target?

Comment (jafcrocker) [inline servicemigration/context.py]: This change was simply to enable running the tests.  It was the only change in the [Fix unit tests] (https://github.com/control-center/service-migration/pull/44/commits/e09edacfbfc09056ccb8f91cfeb04abe364ca2be) commit.  
The tests were broken a [few weeks ago](https://github.com/control-center/service-migration/pull/43); nobody noticed until now.
